test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -346,13 +340,10 @@ describe('Transfer custom tokens @no-mmi', function () {
 
           // check token amount is correct after transaction
           await clickNestedButton(driver, 'Tokens');
-          const tokenAmount = await driver.findElement(
-            {
-              css: '[data-testid="multichain-token-list-item-value"]',
-              text: '8.5 TST',
-            },
-            { timeout: 10000 },
```

Comment:
remove the timeout as the default timeout for `findElement` is 10000, we don't need to specify here